index.html

```diff
@@ -289,6 +289,93 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer>
+        <section class="contact">
+          <form id="form" action="https://formspree.io/f/mlezobkj" method="POST">
+            <div id="contact-section" class="container-contact">
+              <p class="description-contact">
+                I'm always interested in hearing about new projects, so if you'd
+                like to chat please get in touch.
+              </p>
+              <ul class="contact-inputs">
+                <li class="form-field">
+                  <input
+                    id="full-name"
+                    name="full-name"
+                    type="text"
+                    maxlength="30"
+                    placeholder="Full Name"
+                    required
+                  />
+                </li>
+                <li class="form-field">
+                  <input
+                    id="email-address"
+                    name="email-address"
+                    type="email"
+                    placeholder="Email address"
+                    required
+                  />
+                  <small></small>
+                </li>
+                <li class="form-field">
+                  <textarea
+                    id="text-area"
+                    name="text-area"
+                    maxlength="500"
+                    placeholder="Enter your text here"
+                    required
+                  ></textarea>
+                </li>
+              </ul>
+            </div>
+            <div class="container-button">
+              <button class="button-submit" type="submit">Get in touch</button>
+            </div>
+          </form>
+        </section>
+        <hr class="top-line-social-icons" />
+        <div class="container-icons-footer">
+          <a target="_blank" href="">
+            <img
+              class="social-icons-footer"
+              src="./Icons/Git-Hub-icon-Footer.svg"
+              alt="Git-Hub-icon-Footer"
+            />
+          </a>
+          <a target="_blank" href="">
+            <img
+              class="social-icons-footer"
+              src="./Icons/Disabled.svg"
+              alt="Linkedin-icon-Footer"
+            />
+          </a>
+          <a href="#">
+            <img
+              class="social-icons-footer"
+              src="./Icons/Icon-number-3-Footer.svg"
+              alt="Git-Hub-icon-Footer"
+            />
+          </a>
+          <a target="_blank" href="">
+            <img
+              class="social-icons-footer"
+              src="./Icons/Twitter-icon-Footer.svg"
+              alt="Twitter-icon-Footer"
+            />
+          </a>
+          <a target="_blank" href="">
+            <img
+              class="social-icons-footer"
+              src="./Icons/Microverse-Icon-Footer.svg"
+              alt="Microverse-icon-Footer"
+            />
+          </a>
+        </div>
```

Comment:
Nice job on the footer layout! But, the links to your images need to be fixed 🔧. It's important to display the correct images inside of your project so all recruiters can see the logos of all your social profiles and head to them easily!

![image](https://user-images.githubusercontent.com/97407840/197102273-37e435da-d8bb-4550-8321-db8f3e15b760.png)

Please, change the [`src` attribute](https://www.w3schools.com/Tags/att_img_src.asp) to the corresponding path of each image inside of the `media` folder.

They should look as the following:

![image](https://user-images.githubusercontent.com/97407840/197102345-b037085a-ae79-4819-8298-aa81ce0c254e.png)